packages/sol-rayz/src/getParsedAccountByMint.ts

```diff
@@ -0,0 +1,27 @@
+// @ts-nocheck
+import { clusterApiUrl, Connection, Commitment, PublicKey } from '@solana/web3.js';
+import { TOKEN_PROGRAM } from './config/solana';
+
+export const createConnectionConfig = (clusterApi = clusterApiUrl('mainnet-beta'), commitment = 'confirmed') =>
+  new Connection(clusterApi, commitment as Commitment);
```

Comment:
this code duplicate next file
packages/sol-rayz/src/getParsedNftAccountsByCollection.ts

probably better to move to `utils` or something